src/utils/downloadFile.ts

```diff
@@ -36,3 +36,19 @@ export function downloadFileFromURL(downloadURL: string, fileName: string) {
       throw new Error('Failed to download file');
     });
 }
+
+export function blobToBase64(fileBlob: File): Promise<string> {
+  return new Promise((resolve, _) => {
+    const reader = new FileReader();
+    reader.onloadend = () => resolve((reader.result as string).split(',')[1]);
```

Comment:
It'd probably be a good idea to put a couple of comments here:
* the type assertion `reader.result as string` is safe because we use `readAsDataURL()` - see https://developer.mozilla.org/en-US/docs/Web/API/FileReader/result#value.
* the `.split(',')[1]` code is necessary to remove the Data URL prefix added by `reader.readAsDataURL()` - see note on https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL.

Reply:
Speaking of comments, it might be a good idea to put a comment somewhere in the frontend code talking about _why_ we're doing this, that we're using base64 to avoid false positives from the WAF, and mention that ticket EASI-3075 has more details.

Reply:
Not the biggest deal but to be clearer maybe `blobToBase64` should be named `getFileBlobAsBase64` or `fileBlobToBase64`, and arg `fileBlob: File` -> `file: File`.

Reply:
implemented suggestions, thank you both!